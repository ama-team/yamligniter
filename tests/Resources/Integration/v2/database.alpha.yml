active_group: main
db:
  main:
    hostname: mysql.srv01.company.name
    username: betsy
    password: secure
    database: codeigniter
  failover:
    password: root
    database: codeigniter
