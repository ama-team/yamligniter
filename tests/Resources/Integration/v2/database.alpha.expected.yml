active_record: true
active_group: main
db:
  main:
    hostname: mysql.srv01.company.name
    username: betsy
    password: secure
    database: codeigniter
    dbdriver: mysql
    dbprefix: ''
    pconnect: true
    db_debug: false
    cache_on: false
    cachedir: ''
    char_set: utf8
    dbcollat: utf8_general_ci
    swap_pre: ''
    autoinit: true
    stricton: false
  failover:
    hostname: localhost
    username: root
    password: root
    database: codeigniter
    dbdriver: mysql
    dbprefix: ''
    pconnect: true
    db_debug: false
    cache_on: false
    cachedir: ''
    char_set: utf8
    dbcollat: utf8_general_ci
    swap_pre: ''
    autoinit: true
    stricton: false
